src/test/resources/environment-it-sfdc-mailchimp.json

```diff
@@ -0,0 +1,41 @@
+{
+  "apiKey": "abc123",
+
+  "crmPrimary": "salesforce",
+
+  "salesforce": {
+    "username": "brett@impactupgrade.com",
+    "password": "4mJA6UrqffKZEP35cWnrqxGmeZJGCi2IFHM2P5ag",
+
+    "fieldDefinitions": {
+      "emailOptOut": "HasOptedOutOfEmail",
+      "emailBounced": "",
+      "paymentGatewayName": "Payment_Gateway_Name__c",
+      "paymentGatewayTransactionId": "Payment_Gateway_Transaction_Id__c",
+      "paymentGatewayCustomerId": "Payment_Gateway_Customer_Id__c",
+      "paymentGatewaySubscriptionId": "Payment_Gateway_Subscription_Id__c"
```

Comment:
Picky: you can leave off the paymentGateway stuff if you want. Not needed in this one.